help/style.css

```diff
@@ -0,0 +1,109 @@
+*{
```

Comment:
Do you really need all this stuff? Your page will already include my Bootstrap css files (as you can see by viewing the source on sample.php) and this page should generally match the style of the rest of the site.